package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "gatsby-ready-to-go",
+  "private": true,
+  "version": "0.1.0",
+  "description": "Everything you need to create an amazing Gatsby project.",
+  "author": "Pagepro",
+  "license": "0BSD",
+  "scripts": {
+    "build": "gatsby build",
+    "develop": "gatsby develop",
+    "start": "npm run develop",
+    "serve": "gatsby serve",
+    "clean": "gatsby clean",
+    "lint": "eslint --ignore-path .gitignore \"src/**/*.+(ts|js|tsx)\"",
+    "plop": "plop",
+    "storybook": "start-storybook -p 6006",
+    "build:storybook": "build-storybook -c .storybook -o storybook-build -s public",
+    "build-storybook": "build-storybook"
+  },
+  "dependencies": {
+    "@emotion/core": "^10.0.35",
+    "@emotion/styled": "^10.0.27",
+    "gatsby": "^2.31.1",
```

Comment:
Unfortunately, we can't do this for now because plugin gatsby-plugin-theme-UI is not compatible with gatsby version 3.0.3 - It requires gatsby@^2.13.1

![image](https://user-images.githubusercontent.com/18337813/110238700-5b865a00-7f43-11eb-86f0-44c45616985b.png)

Reply:
Upgraded to Gatsby v3.